pkg/crc/services/dns/dns_darwin.go

```diff
@@ -18,12 +18,14 @@ import (
 
 const (
 	resolverFileTemplate = `port {{ .Port }}
+domain {{ .Domain }}
```

Comment:
Did you test this with a filename != domain name ? Only documentation about the format of that file which I found is https://hints.macworld.com/article.php?story=2004062902195410 which is very old, but says that `domain` must be the first entry in the file.

Reply:
@cfergeau yes tested it and it does work, also in the article I just notice it mentioned `you can use any name if you specify the option domain foobar.com as the first one in that file` not first entry in the file ( not sure if I missed something )

```
$ ls /etc/resolver/
foo

17:52 $ cat /etc/resolver/foo
port 53
domain testing
nameserver 192.168.64.92
search_order

17:52 $ ping -c 4 bha.apps-crc.testing
PING bha.apps-crc.testing (192.168.64.92): 56 data bytes
64 bytes from 192.168.64.92: icmp_seq=0 ttl=64 time=0.222 ms
64 bytes from 192.168.64.92: icmp_seq=1 ttl=64 time=0.259 ms
64 bytes from 192.168.64.92: icmp_seq=2 ttl=64 time=0.461 ms
64 bytes from 192.168.64.92: icmp_seq=3 ttl=64 time=0.298 ms

--- bha.apps-crc.testing ping statistics ---
4 packets transmitted, 4 packets received, 0.0% packet loss
round-trip min/avg/max/stddev = 0.222/0.310/0.461/0.091 ms

```